whistl/label_samples.py

```diff
@@ -71,7 +71,7 @@ def edit_fields(fields, sample_data):
         choice = input('Add/remove/done: ').strip().lower()
 
         if choice == 'add':
-            print('Which field would you like to remove?')
+            print('Which field would you like to add?')
```

Comment:
Just to clarify, is this an option where a user can specify a field (i.e. column in the metadata file) that contains their label (for example a "diagnosis" column)?

Reply:
By default the script only prints the title for each sample. If you want to see more metadata it gives the options `info`, which prints all the metadata for the current sample or `field` which allows you to add or remove columns in the metadata file that are printed by default. 

Sorry that it's unclear here. The explanation of what a field is is printed in the prompt for each sample and also in the function comment. Neither are in the Github diff, so I should have given some context in the PR body.